target/linux/ath79/config-4.14

```diff
@@ -79,6 +79,8 @@ CONFIG_GPIO_74X164=y
 CONFIG_GPIO_ATH79=y
 CONFIG_GPIO_GENERIC=y
 CONFIG_GPIO_SYSFS=y
+CONFIG_GPIO_WATCHDOG=y
```

Comment:
Thanks to 1724d4e9ed7716a5444eca42adf045c2f3afc554. You should be able to drop this config-4.14 update altogether.